docs/content.zh/release-notes/flink-1.14.md

```diff
@@ -0,0 +1,424 @@
+API.
+
+#### Deprecate FlinkKafkaConsumer
+
+##### [FLINK-24055](https://issues.apache.org/jira/browse/FLINK-24055)
+
+FlinkKafkaConsumer has been deprecated in favor of KafkaSource, FlinkKafkaProducer has been
+deprecated in favor of KafkaSink.
+
+#### Pulsar Source
+
+##### [FLINK-20731](https://issues.apache.org/jira/browse/FLINK-20731)
+
+Flink now directly provides a way to read data from Pulsar with DataStream API.
+
+#### Add Support for Azure Data Lake Store Gen 2 in Flink File System
+
+##### [FLINK-18562](https://issues.apache.org/jira/browse/FLINK-18562)
+
+Flink now supports reading from Azure Data Lake Store Gen 2 with the flink-azure-fs-hadoop
+filesystem using the abfs(s) scheme.
+
+#### InputStatus should not contain END_OF_RECOVERY
+
+##### [FLINK-23474](https://issues.apache.org/jira/browse/FLINK-23474)
+
+`InputStatus.END_OF_RECOVERY` was removed. It was an internal flag that should never be returned from
+SourceReaders. Returning that value in earlier versions might lead to misbehaviour.
+
+#### Connector-base exposes dependency to flink-core.
+
+##### [FLINK-22964](https://issues.apache.org/jira/browse/FLINK-22964)
+
+Connectors do not transitively hold a reference to `flink-core` anymore. That means that a fat jar
+with a connector does not include `flink-core` with this fix.
+
+### Runtime & Coordination
+
+#### Increase akka.ask.timeout for tests using the MiniCluster
+
+##### [FLINK-23906](https://issues.apache.org/jira/browse/FLINK-23906)
+
+The default `akka.ask.timeout` used by the `MiniCluster` has been increased to 5 minutes. If you
+want to use a smaller value, then you have to set it explicitly in the passed configuration.
+
+#### The node IP obtained in NodePort mode is a VIP
+
+##### [FLINK-23507](https://issues.apache.org/jira/browse/FLINK-23507)
+
+When using `kubernetes.rest-service.exposed.type=NodePort`, connection string for Rest gateway is
+now correctly constructed in form of `<nodeIp>:<nodePort>` instead of
+`<kubernetesApiServerUrl>:<nodePort>`. This may be a breaking change for some users.
+
+This also introduces a new config option `kubernetes.rest-service.exposed.node-port-address-type` that
+lets you select `<nodeIp>` from a desired range.
+
+#### Timeout heartbeat if the heartbeat target is no longer reachable
+
+##### [FLINK-23209](https://issues.apache.org/jira/browse/FLINK-23209)
+
+Flink now supports to detect dead TaskManagers via the number of consecutive failed heartbeat RPCs.
+The threshold until a TaskManager is marked as unreachable can be configured
+via `heartbeat.rpc-failure-threshold`. This can speed up the detection of dead TaskManagers
+significantly.
+
+#### RpcService should fail result futures if messages could not be sent
+
+##### [FLINK-23202](https://issues.apache.org/jira/browse/FLINK-23202)
+
+Flink now fails rpc requests that cannot be delivered immediately instead of waiting for
```

Comment:
Similarly as above, I would slightly adjust this to:

The same way Flink detects unreachable heartbeat targets faster, Flink now also immediately fails RPCs where the target is known by the OS to be unreachable on a network level, instead of waiting for a timeout (`akka.ask.timeout`).

One effect this are faster task failovers, because cancelling tasks on a dead `TaskExecutor` no longer gets delayed by the RPC timeout.

If this faster failover is a problem in certain setups (which might rely on the fast that external systems hit timeouts), we recommend to configure the application's restart strategy with a restart delay.

Reply:
@tillrohrmann Could you double-check the proposal from Stephan? I can apply the change later.

Reply:
Sounds good 👍